audiophiler/templates/main.html

```diff
@@ -19,6 +19,12 @@
             </div>
         {% endif %}
 
+        {% if (file.author == auth_dict["uid"] or is_rtp or is_eboard) and is_admin %}
```

Comment:
Does the file variable exist here? It looks like it's defined by the for loop on line 28 (`{% for file in db_files%}`)

Reply:
I see what you mean I'll change this. Must have skipped my mind